### DISCUSSION TITLE
Steam++无法使用Steam账号登录/无法绑定Steam账号，Gateway 504

### DISCUSSION THREAD

**Discussion by PriceHu** [BeyondDimension/SteamTools#1010] [General]
RT，登录时经常会显示`504 Gateway Time-out`，重复登录尝试（取消后登录或点击重试），有概率可以成功登录。

- 程序版本(Program Version): 2.6.7
- 系统版本号(OS Version): example Windows 10.19042.1526
- 是否为应用商店版本(Whether it is a Store version): 否
- 所在地区：CA
- 是否开启任何代理或加速服务：否

不知道服务器的逻辑是怎样的，登录流程上有极大的概率504，但是在浏览器刷新请求，~甚至于用postman带着空参数连续尝试数次~，状态都是200。

**Comment by Kal-tsit63** [comment]
相同的问题，而且更严重，尝试十几次以上无一次成功
程序版本2.8.2
不是应用商店版本

**Comment by rmbadmin** [comment]
这是服务端问题，等待服务端更新修复即可
